pylama.ini

```diff
@@ -19,7 +19,11 @@ ignore = E1101
 # E1101 Instance of 'Resource' has no 'instances' member [pylint]
 
 [pylama:cloudmarker/stores/mongodbstore.py]
-ignore = R0913 
+ignore = R0913
+# R0913 Too many arguments (7/5)
+
+[pylama:cloudmarker/stores/emailstore.py]
```

Comment:
`store` -> `alert` & `cloudmarker/stores/` ->`cloudmarker/alert/`

Reply:
A little confusion here. Do you want alerts to be in the separate package ?